### PR TITLE
performance(loading) - experimentally remove initial-load.css from index

### PR DESCRIPTION
**Pull request by maltenuhn**
**Problem:**
We possibly block first render on completion of loading of `initial-load.css`

**Fix:**
Move this to the list of React-loaded css files, and remove it from index.html. 

(also remove a prefetch for an image that never executes in time and leads to browser warning)

### CODE REVIEW

**Review by balazsbajorics — APPROVED** [review]

File: editor/src/templates/editor-entry-point.tsx [hunk 1]
```diff
@@ -19,9 +19,11 @@ import { addStyleSheetToPage } from '../core/shared/dom-utils'
 import { STATIC_BASE_URL } from '../common/env-vars'
 
 const editorCSS = [
-  `${STATIC_BASE_URL}editor/css/canvas.css`,
-  `${STATIC_BASE_URL}editor/css/slider.css`,
-  `${STATIC_BASE_URL}editor/css/ReactContexify.css`,
+  `${STATIC_BASE_URL}editor/initial-load.css`,
+  `${STATIC_BASE_URL}editor/canvas.css`,
+  `${STATIC_BASE_URL}editor/slider.css`,
+  `${STATIC_BASE_URL}editor/cursors.css`,
```
Comment (balazsbajorics) [inline editor/src/templates/editor-entry-point.tsx:25]: seems like someone deleted cursors.css from the editor-entry-point recently?

Comment (balazsbajorics) [reply editor/src/templates/editor-entry-point.tsx:25]: @maltenuhn it was you.

Comment (maltenuhn) [reply editor/src/templates/editor-entry-point.tsx:25]: Ah yes. It was already empty.

**Review by balazsbajorics — APPROVED** [review]

File: editor/src/templates/editor-entry-point.tsx [hunk 2]
```diff
@@ -19,9 +19,10 @@ import { addStyleSheetToPage } from '../core/shared/dom-utils'
 import { STATIC_BASE_URL } from '../common/env-vars'
 
 const editorCSS = [
-  `${STATIC_BASE_URL}editor/css/canvas.css`,
-  `${STATIC_BASE_URL}editor/css/slider.css`,
-  `${STATIC_BASE_URL}editor/css/ReactContexify.css`,
+  `${STATIC_BASE_URL}editor/initial-load.css`,
+  `${STATIC_BASE_URL}editor/canvas.css`,
+  `${STATIC_BASE_URL}editor/slider.css`,
+  `${STATIC_BASE_URL}editor/ReactContexify.css`,
```
Comment (balazsbajorics) [inline editor/src/templates/editor-entry-point.tsx:25]: agh, you tricked my eyes. you are pointing these to `editor/` but on master they point to `editor/css/`

Comment (maltenuhn) [reply editor/src/templates/editor-entry-point.tsx]: fixed!!

**Review by balazsbajorics — APPROVED** [review]

**Comment by balazsbajorics** [comment]
I'm gonna merge this, as the Link to test editor is already showing a huge difference in loading time

**Review by Rheeseyb — COMMENTED** [review]
:+1:

**Pull request merged**